content/globals.d.ts

```diff
@@ -1,11 +1,3 @@
-// https://stackoverflow.com/questions/39040108/import-class-in-definition-file-d-ts
-type $FolderImport = import('./folder-import').$FolderImport
-declare var FolderImport: $FolderImport // eslint-disable-line no-var
-declare const Zotero: { FolderImport: $FolderImport } & Omit<Record<string, any>, 'FolderImport'>
-declare const AddonManager: any
-declare const Components: any
-declare const PathUtils: any
-declare const IOUtils: any
-declare const Services: any
-declare const Localization: any
-declare const rootURI: string
+declare namespace Zotero {
+  let FolderImport: import('./folder-import').$FolderImport
```

Comment:
Can extend any attributes under the ns `Zotero`

Reply:
What does `FolderImport` mean?

Reply:
It's the instance created by this plugin